Pipfile

```diff
@@ -6,7 +6,7 @@ name = "pypi"
 [packages]
 pkginfo = "*"
 twine = "*"
-pip = "==23.0.1"
+pip = "==24.2"
```

Comment:
I would be surprised if this doesn't break anything ...

The get step is usually very sensitive on this... On the other hand, since there is no pipeline using this resource to do a get step, and we block the implicit get... we care only that put is working.... Anyway, no matter our usage of this resource in Pix4D I would still check that this is not breaking the get step by running integration tests with nexus...

Also, double check if this fixes: https://pix4dbug.atlassian.net/browse/PCI-3756
so we can close also that ticket..